rplugin/python3/deoplete/source/omni.py

```diff
@@ -61,11 +61,13 @@ def _get_complete_position(self, context, current_ft, filetype):
                                            'Manual' and m is None):
                     continue
 
-                if filetype == current_ft and self._omnifunc in [
+                if ((filetype == current_ft or
+                        filetype in ['css', 'javascript'])
```

Comment:
Alternatively remove filetype condition (remove `((filetype == current_ft or filetype in ['css', 'javascript'])`) and rely only on blacklisted omnifunc's.

Reply:
I think specify omnifuncs names are better.